client/scss/components/_chooser.scss

```diff
@@ -64,6 +64,21 @@
   @apply w-body-text-large;
 }
 
+@media (hover: hover) {
+  .chooser__actions {
+    opacity: 0;
+  }
+
+  .w-field__input:hover,
+  .w-field__input:focus-within {
+    // Hiding for devices capable of hover states only,
+    // with opacity only so keyboard focus can move to the first interactive element to reveal it.
```

Comment:
I think this comment would be more helpful if it was above `@media (hover: hover) {` like before, so we understand why we need the media query.